plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeLocalConcurrentWritesTest.java

```diff
@@ -356,13 +356,13 @@ public void testConcurrentInsertsReconciliationForMixedInserts()
                     "SELECT * FROM " + tableName,
                     "VALUES (0, 10), (1, 10), (11, 20), (1, 20), (22, 30)");
             assertQuery(
-                    "SELECT version, operation, isolation_level, read_version FROM \"" + tableName + "$history\"",
+                    "SELECT operation, isolation_level, is_blind_append FROM \"" + tableName + "$history\"",
                     """
                             VALUES
-                                (0, 'CREATE TABLE AS SELECT', 'WriteSerializable', 0),
-                                (1, 'WRITE', 'WriteSerializable', 0),
-                                (2, 'WRITE', 'WriteSerializable', 1),
-                                (3, 'WRITE', 'WriteSerializable', 2)
+                                ('CREATE TABLE AS SELECT', 'WriteSerializable', true),
+                                ('WRITE', 'WriteSerializable', false),
+                                ('WRITE', 'WriteSerializable', false),
+                                ('WRITE', 'WriteSerializable', true)
```

Comment:
Why were operation and read_version columns removed?

Reply:
`operation` and `read_version` were removed because we can't say assert with certainty which of the operations is the blind insert